### PR TITLE
New package: psfiles 0.1.0

### PR DESCRIPTION
**Pull request by mukovnin**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/psfiles/template [hunk 1]
```diff
@@ -0,0 +1,11 @@
+pkgname=psfiles
+version=0.1.0
+revision=1
+archs="x86_64"
```
Comment (Chocimier) [inline srcpkgs/psfiles/template:4]: remove archs restriction even if you can't test, let CI build it.

Comment (Chocimier) [reply srcpkgs/psfiles/template:4]: Upstream support only x86_64. Okay, enable musl by changing to x86_64* and leave comment.

Comment (mukovnin) [reply srcpkgs/psfiles/template]: @Chocimier fixed

**Comment by mukovnin** [comment]
xlint warnings fixed

**Comment by melroy89** [comment]
Why close it?

**Pull request closed without merging**